packages/playground/src/screens/home.tsx

```diff
@@ -5,29 +5,30 @@ import { ImageURISource, ScrollView } from 'react-native'
 import { useLink } from 'solito/link'
 import { Button, Image, Separator, Text, View, XStack, YGroup, YStack } from 'tamagui'
 
-import { Connection, useConnections } from '../provider/storage'
+import { BUILTIN_CONNECTIONS, Connection, useConnections } from '../provider/storage'
 
 export function HomeScreen() {
   const [state] = useConnections()
   return (
     <ScrollView>
       <YStack padding="$4" gap="$4">
         <HeroImage />
-        <YGroup separator={<Separator />}>
-          {state.connections.map((connection) => (
-            <ConnectionItem key={connection.id} connection={connection} />
-          ))}
-        </YGroup>
-        <NewConnectionButton />
-        {/* <Button
-          color="$green10"
-          chromeless
-          onPress={() => {
-            addConnection({ host: 'ws://10.255.1.59:3990', path: '', label: 'Test Connection' })
-          }}
-        >
-          TEST - New Connection
-        </Button> */}
+        <YStack gap="$2">
+          <YGroup separator={<Separator />}>
+            {state.connections.map((connection) => (
+              <ConnectionItem key={connection.id} connection={connection} />
+            ))}
+          </YGroup>
+          <NewConnectionButton />
+        </YStack>
+        <YStack gap="$2">
+          <Text textAlign="center">or try some of the examples below:</Text>
+          <YGroup separator={<Separator />}>
+            {Object.values(BUILTIN_CONNECTIONS).map((connection) => (
```

Comment:
I think we should hide these once you have added your own connection

Reply:
Maybe swap with a button so you can still play around? Otherwise won't ever be able to check them out!

Reply:
Sure, that could work. But once you have added your own server you probably don't really care about the examples. And you could always follow a URL to open them from the website